gulpfile.js

```diff
@@ -73,7 +75,31 @@ gulp.task('styles:toolkit', function () {
 		.pipe(gulpif(config.dev, reload({stream:true})));
 });
 
-gulp.task('styles', ['styles:fabricator', 'styles:toolkit']);
+gulp.task('styles:toolkit_base', function () {
```

Comment:
@slowbot There's no need for duplicate tasks. You can give `gulp.src` an array, like `[config.src.styles.toolkit_base, config.src.styles.toolkit_lap, config.src.styles.toolkit]` and it should output all three files.